wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -479,4 +480,55 @@ class Wallet(
       throw e
     }
   }
+
+  /**
+   * Get account operations for the specified Stellar address.
+   *
+   * @param accountAddress Stellar address of the account
+   * @param limit how many operations to fetch, maximum is 200
+   * @param order optional data order, ascending or descending, defaults to descending
+   * @param cursor optional cursor to specify a starting point
+   * @param includeFailed optional flag to include failed operations, defaults to false
+   *
+   * @return a list of formatted operations
+   *
+   * @throws [OperationsLimitExceededException] when maximum limit of 200 is exceeded
+   */
+  suspend fun getHistory(
+    accountAddress: String,
+    limit: Int,
+    order: Order? = Order.DESC,
+    cursor: String? = "",
+    includeFailed: Boolean? = false
+  ): List<WalletOperation> {
+    if (limit > 200) {
+      throw OperationsLimitExceededException()
+    }
+
+    val orderValue =
+      if (order == Order.ASC) {
+        RequestBuilder.Order.ASC
+      } else {
+        RequestBuilder.Order.DESC
+      }
+
+    val accountOperations =
+      server
+        .operations()
+        .forAccount(accountAddress)
+        .limit(limit)
+        .order(orderValue)
+        .cursor(cursor)
+        .includeFailed(includeFailed!!)
```

Comment:
Could you please explain why this could lead to a potential null pointer exception? My thinking is that since the default value is `false` and the accepted type is `boolean`, it should never be `null`.

Reply:
:nerd_face: `Boolean?` in Kotlin means that it can accept `null` as well. It's not optional like it is in Typescript. Will fix.